packages/common/src/components/utils/StatusWithIcon.tsx

```diff
@@ -7,7 +7,8 @@ const Wrapper = styled.div`
   display: flex;
   align-items: center;
 
-  .statusContent > a {
+  .statusContent > a,
```

Comment:
In general I think we should avoid styles that "reach into" child components. Alternatives could be to keep contextual styles within the child component (if they're of a general kind) eg.
```typescript
import ContextualComponent from './ContextualComponent'

const Wrapper = styled.div`
  // styles

  ${ContextualComponent} & {
    // contextual styles
  }
`
```
it makes it easier to understand how the component is used and eases refactoring

Reply:
The main idea is that the component that needs contextual styles (`<a />`) owns it's own styles. It's kind of like Inversion of Control in OO. In this case it might not matter, or even be applicable, but I wanted to introduce the suggestion as I think it will make the code easier to reason about and refactor.